src/v/redpanda/admin_server.h

```diff
@@ -146,28 +147,32 @@ class admin_server {
           _server._routes,
           [this, handler](std::unique_ptr<ss::http::request> req)
             -> ss::future<ss::json::json_return_type> {
-              auto auth_state = apply_auth<required_auth>(*req);
```

Comment:
This was a fun footgun if we ever accessed this after the first suspension point :)